amd/src/package_search/mutations.js

```diff
@@ -98,7 +99,10 @@ export default class {
             stateManager.setReadOnly(false);
             // Update category specific data.
             for (const [index, category] of categories.entries()) {
-                state[category].data = await results[index];
+                const result = await results[index];
+                state[`${category}Packages`] = result.packages;
```

Comment:
Könntest du die packages nicht auch in `state[category].packages` speichern?

Reply:
Leider wären die Pakete dann immernoch zu tief im Objekt, sodass keine Events, z.B. bei der Modifizierung der `isfavourite` property, getriggert werden.